### PR TITLE
Add instructions for running locally

### PR DESCRIPTION
**Pull request by jcollins-g**
@kevmoo

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -37,3 +37,30 @@ Contributions made by corporations are covered by a different agreement than the
...
    and submit a PR for review.
+
+## How to run DartPad locally
+
+* To run the DartPad against the regular serving backend:  `grind serve`.  This serves the
+DartPad frontend locally on port 8000.
+
+* To run DartPad against a local version of the dart-services backend:
+```bash
+pub global activate grinder
```
Comment (kevmoo) [inline CONTRIBUTING.md]: This should move above – since the first bullet mention using `grind serve`

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -37,3 +37,30 @@ Contributions made by corporations are covered by a different agreement than the
  
 3) Add or change sample Gist IDs to the [index file](https://github.com/dart-lang/dart-pad/blob/master/web/index.html#L54),
    and submit a PR for review.
+
+## How to run DartPad locally
+
+* To run the DartPad against the regular serving backend:  `grind serve`.  This serves the
```
Comment (kevmoo) [inline CONTRIBUTING.md]: I crashed because I don't have `vulcanize` installed? Do we need to list dependencies?

Comment (kevmoo) [reply CONTRIBUTING.md]: looks like we need node v6, too?

Comment (jcollins-g) [reply CONTRIBUTING.md]: We require node for vulcanize, yes.

Comment (kevmoo) [reply CONTRIBUTING.md]: Put that up top? Tell folks they need vulcanize + node 6 installed.

File: CONTRIBUTING.md (hunk 1 above)
Comment (kevmoo) [reply CONTRIBUTING.md]: Also can mention that it runs fine with `pub run build_runner serve` – and reloads a lot faster on updates, too!

Comment (jcollins-g) [reply CONTRIBUTING.md]: @kevmoo It's great that that works, but given that there are a lot of hand-hacked bits in the grinder script for how to build this thing, I'm reluctant to recommend that method.  grind serve is guaranteed to get you what would be deployed.

File: CONTRIBUTING.md (hunk 2 above)
Comment (jcollins-g) [reply CONTRIBUTING.md]: npm v5 is actually fine too.  Will mention this.

File: CONTRIBUTING.md (hunk 1 above)
Comment (jcollins-g) [reply CONTRIBUTING.md]: done

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -37,3 +37,37 @@ Contributions made by corporations are covered by a different agreement than the
  
 3) Add or change sample Gist IDs to the [index file](https://github.com/dart-lang/dart-pad/blob/master/web/index.html#L54),
    and submit a PR for review.
+
+## How to run DartPad locally
+
+DartPad requires npm v5 or higher and vulcanize to be installed.
```
Comment (kevmoo) [inline CONTRIBUTING.md]: It seems like node v6 is required – not greater. At least on my machine...

Comment (jcollins-g) [reply CONTRIBUTING.md]: Ah.  My Linux box seems to work ok with npm v5.. maybe it depends.

**Pull request merged**